db/migrate/20170202195228_migrate_ansible_tower_configuration_manager_settings_to_automation_manager.rb

```diff
@@ -0,0 +1,45 @@
+class MigrateAnsibleTowerConfigurationManagerSettingsToAutomationManager < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    serialize :value
+  end
+
+  def up
+    say_with_time('Migrate /ems_refresh/ansible_tower_configuration% to be of Automation Manager') do
+      SettingsChange.where('key LIKE ?', '/ems_refresh/ansible_tower_configuration%').each do |s|
+        s.key = s.key.sub(%r{/ansible_tower_configuration}, '/ansible_tower_automation')
```

Comment:
Why do you have a regex here and on line 18?  They're not needed, and a normal string will suffice.

```ruby
[1] pry(main)> "abcdefg".sub("def", "xxx")
=> "abcxxxg"
```